src/looprestoration_tmpl_16.rs

```diff
@@ -543,7 +334,32 @@ unsafe extern "C" fn padding(
         }
     };
 }
+
 unsafe extern "C" fn wiener_c(
+    mut p: *mut libc::c_void,
```

Comment:
For the shim functions, the `mut`s on args will all be unused so we can remove them.